esb-connector-creatio-crm/src/test/java/org/wso2/carbon/connector/integration/test/creatiocrm/CreatioCRMRestConnectorIntegrationTest.java

```diff
@@ -0,0 +1,261 @@
+
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.connector.integration.test.creatiocrm;
+
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.connector.integration.test.base.ConnectorIntegrationTestBase;
+import org.wso2.connector.integration.test.base.RestResponse;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class CreatioCRMRestConnectorIntegrationTest extends ConnectorIntegrationTestBase {
+    private final Map<String, String> eiRequestHeadersMap = new HashMap<String, String>();
+
+    private final Map<String, String> apiRequestHeadersMap = new HashMap<String, String>();
+
+    private String apiUrlEndPoint;
+
+    /**
+     * Set up the environment.
+     */
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+
+        String connectorName = System.getProperty("connector_name") + "-connector-" +
+                System.getProperty("connector_version") + ".zip";
+        addCertificatesToEIKeyStore("client-truststore.jks", "wso2carbon");
+        init(connectorName);
+        eiRequestHeadersMap.put("Accept-Charset", "UTF-8");
+        eiRequestHeadersMap.put("Content-Type", "application/json");
+    }
+
+    @Test(groups = {"wso2.ei"}, description = "Authentication for REST APIs")
+    public void authenticationWithMandatoryParameters() throws Exception {
+        String methodName = "postAuth";
+        RestResponse<JSONObject> esbRestResponse = sendJsonRestRequest(getProxyServiceURLHttp(methodName),
+                "POST", eiRequestHeadersMap, null);
+        Assert.assertEquals(esbRestResponse.getHttpStatusCode(), 200);
+        Assert.assertTrue(esbRestResponse.getHeadersMap().containsKey("BPMCSRF"));
+        Assert.assertTrue(esbRestResponse.getHeadersMap().containsKey("Cookie"));
+    }
+
+
```

Comment:
Shall we remove these unnecessary empty lines?

Reply:
Removed